script.js

```diff
@@ -1,67 +1,104 @@
+/* eslint-disable max-classes-per-file */
+
 class Books {
   constructor(title, author) {
     this.title = title;
     this.author = author;
+    this.bookShelf = [];
   }
 }
 
 const buttonAdd = document.querySelector('.button-add');
 const displayBooks = document.getElementById('books-list');
-const bookShelf = [];
 
-// Storage functions
+// Storage classes
+class Store {
+  static storeGet() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
```

Comment:
- [ ] After refactoring to use JavaScript `classes`, please let's ensure to keep all functionality of the application. so that the refactoring did not introduce bugs to the app and all functions from the previous project are still working fine.

- [ ] Please kindly fix the issue so that your app data are preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md). 🛠💪